paimon-common/src/main/java/org/apache/paimon/utils/BitSliceIndexRoaringBitmap.java

```diff
@@ -220,6 +309,7 @@ public boolean isNotEmpty() {
         public void serialize(DataOutput out) throws IOException {
             out.writeByte(VERSION_1);
             out.writeLong(min);
+            out.writeLong(max);
```

Comment:
Can you add detailed format to the page: https://paimon.apache.org/docs/master/concepts/spec/fileindex/

You can add an explanation for `bsi serialized`.